inlong-agent/agent-plugins/src/test/java/org/apache/inlong/agent/plugin/TestFileAgent.java

```diff
@@ -144,6 +146,19 @@ public void testOneJobOnly() throws Exception {
         Assert.assertTrue(checkOnlyOneJob());
     }
 
+    // @Test
+    public void testOneJobFullPath() throws Exception {
```

Comment:
the test cannot test full mode

Reply:
The trigger will be submitted in the method, and the full file will be read